app/ts/schema.ts

```diff
@@ -23,21 +42,15 @@ export const TokenSchema = funtypes.Object({
 	decimals: BigIntHexSchema,
 })
 
-export const TokenListSchema = funtypes.Array(TokenSchema)
-
-export const ManagedTokensCacheParserConfig: ParsedValueConfig<funtypes.String, funtypes.Static<typeof TokenListSchema>> = {
-	parse(value) {
-		const jsonParsed = JSON.parse(value)
-		return TokenListSchema.safeParse(jsonParsed)
-	},
-	serialize(value) {
-		const serializedValue = TokenListSchema.safeSerialize(value)
-		if (!serializedValue.success) return { success: false, message: serializedValue.message }
-		const jsonString = JSON.stringify(serializedValue.value)
-		return { success: true, value: jsonString }
-	},
-}
+export type Token = funtypes.Static<typeof TokenSchema>
 
-export const ManagedTokensCacheSchema = funtypes.String.withParser(ManagedTokensCacheParserConfig)
+export const TransferSchema = funtypes.Object({
+	hash: funtypes.String,
+	from: AddressSchema,
+	to: AddressSchema,
+	amount: funtypes.String,
```

Comment:
this should be EthereumQuantity which you can get from interceptors wire types too

Reply:
Creating a separate issue to tread addresses as bigint #200